src/encoded/static/components/genome_browser.js

```diff
@@ -0,0 +1,432 @@
+import React from 'react';
+
+
+const maxFilesBrowsed = 5; // Maximum number of files to browse
+const domainName = 'https://www.encodeproject.org';
+const dummyFiles = [
```

Comment:
This mechanism is probably all out of date. That was for running locally it would just pick some files on production to use.